demos/checkboxradio/icons.html

```diff
@@ -0,0 +1,53 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>jQuery UI Checkboxradio - Icon functionality</title>
+	<link rel="stylesheet" href="../../themes/base/all.css">
+	<script src="../../external/jquery/jquery.js"></script>
+	<script src="../../ui/core.js"></script>
+	<script src="../../ui/widget.js"></script>
+	<script src="../../ui/checkboxradio.js"></script>
+	<link rel="stylesheet" href="../demos.css">
+	<script>
+	$(function() {
+		$( "input" ).checkboxradio({
+			icon: true
```

Comment:
Spec still says default is `false`, but `true` is implemented. Can't remember the discussion, but right now this and the default demo are identical, except for this useless line. If we keep that default, we could rename this to "no-icons.html" and set `icon: false`.